web-app/templates/book/bookListView.html

```diff
@@ -0,0 +1,41 @@
+<table width="100%" class="table table-striped table-bordered table-hover" id="dataTables-example">
+    <thead>
+        <tr style="cursor: pointer">
+            <th ng-click="bC.changeSort('id')">Id
+                <span ng-show="bC.shouldOrderBy('id') && bC.reverse" class="glyphicon glyphicon-menu-up" style="float: right;"></span>
+                <span ng-show="bC.shouldOrderBy('id') && !bC.reverse" class="glyphicon glyphicon-menu-down" style="float: right;"></span>
+            </th>
+            <th ng-click="bC.changeSort('name')">Name
+                <span ng-show="bC.shouldOrderBy('name') && bC.reverse" class="glyphicon glyphicon-menu-up" style="float: right;"></span>
+                <span ng-show="bC.shouldOrderBy('name') && !bC.reverse" class="glyphicon glyphicon-menu-down" style="float: right;"></span>
+            </th>
+            <th ng-click="bC.changeSort('genre')">Genre
+                <span ng-show="bC.shouldOrderBy('genre') && bC.reverse" class="glyphicon glyphicon-menu-up" style="float: right;"></span>
+                <span ng-show="bC.shouldOrderBy('genre') && !bC.reverse" class="glyphicon glyphicon-menu-down" style="float: right;"></span>
+            </th>
+            <th ng-click="bC.changeSort('author')">Author
+                <span ng-show="bC.shouldOrderBy('author') && bC.reverse" class="glyphicon glyphicon-menu-up" style="float: right;"></span>
+                <span ng-show="bC.shouldOrderBy('author') && !bC.reverse" class="glyphicon glyphicon-menu-down" style="float: right;"></span>
+            </th>
+            <th ng-click="bC.changeSort('year')">Year
+                <span ng-show="bC.shouldOrderBy('year') && bC.reverse" class="glyphicon glyphicon-menu-up" style="float: right;"></span>
+                <span ng-show="bC.shouldOrderBy('year') && !bC.reverse" class="glyphicon glyphicon-menu-down" style="float: right;"></span>
+            </th>
+            <th style="width: 5%">Action</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr ng-repeat="book in bC.books | orderBy: bC.sort: bC.reverse | filter: searchText">
+            <td><a href="#/book/{{ book.id }}">{{ book.id }}</a></td>
+            <td>{{ book.name }}</td>
+            <td>{{ book.genre }}</td>
+            <td>{{ book.author.name }}</td>
+            <td>{{ book.year }}</td>
```

Comment:
if **book** is not dynamic, use "{{ :: book.year }}"
One time data-binding **https://toddmotto.com/angular-one-time-binding-syntax**